### PR TITLE
[2.0.x PATCH] Add app.config to net 4.x templates

### PR DESCRIPTION
**Pull request by jbagga**
Cherry picked from https://github.com/aspnet/templating/pull/191 to address https://github.com/aspnet/templating/issues/170

### CODE REVIEW

**Comment by Eilon** [comment]
Why was this closed? Was it merged?

**Comment by Eilon** [comment]
O I C 😄

**Comment by natemcmaster** [comment]
GitHub closed because I renamed branches release/2.0.0 => release/2.0. I retargeted this PR to the new branch.

**Comment by jbagga** [comment]
Tracking Travis failures here https://github.com/aspnet/templating/issues/237

File: src/Microsoft.DotNet.Web.ProjectTemplates/EmptyWeb-FSharp.fsproj.in [hunk 1]
```diff
@@ -10,8 +10,12 @@
     <Compile Include="Program.fs" />
   </ItemGroup>
 
-  <ItemGroup>
+  <ItemGroup Condition="'$(TargetFrameworkOverride)' == ''">
```
Comment (Eilon) [inline src/Microsoft.DotNet.Web.ProjectTemplates/EmptyWeb-FSharp.fsproj.in]: Was this template previously broken in some scenarios? Like on .NET Framework 4.x?

File: test/Templates.Test/EmptyWebTemplateTest.cs [hunk 2]
```diff
@@ -14,7 +14,8 @@ public EmptyWebTemplateTest(ITestOutputHelper output) : base(output)
         }
 
         [ConditionalFact]
-        [OSSkipCondition(OperatingSystems.Linux | OperatingSystems.MacOSX)]
```
Comment (Eilon) [inline test/Templates.Test/EmptyWebTemplateTest.cs]: Is OR'ing these not valid at all? Should we change this attribute to fail if it's used incorrectly?

File: src/Microsoft.DotNet.Web.ProjectTemplates/EmptyWeb-FSharp.fsproj.in (hunk 1 above)
Comment (jbagga) [reply src/Microsoft.DotNet.Web.ProjectTemplates/EmptyWeb-FSharp.fsproj.in]: It was fixed [here](https://github.com/aspnet/templating/pull/191/files#diff-470b05a8dc034a8038d7e404d16ae6b5R15) in dev but was broken before

File: test/Templates.Test/EmptyWebTemplateTest.cs (hunk 2 above)
Comment (jbagga) [reply test/Templates.Test/EmptyWebTemplateTest.cs]: It does not work. Not sure why it was being used. Filed an issue [here](https://github.com/aspnet/Home/issues/2803)

**Comment by natemcmaster** [comment]
Try re-running with this change: https://github.com/aspnet/templating/commit/67fab2fec61121d834dfb9dca359fcc2f47893e6

**Pull request merged**